app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="min-h-screen bg-gray-100 py-6 flex flex-col justify-start sm:py-12">
+  <div class="relative py-3 sm:max-w-xl sm:mx-auto">
+    <div class="relative px-4 py-10 bg-white shadow-lg sm:rounded-3xl sm:p-20">
+      <div class="max-w-md mx-auto">
+        <div>
+          <h2 class="text-2xl font-semibold text-center">Log in</h2>
+        </div>
+        <div class="divide-y divide-gray-200">
+          <div class="py-8 text-base leading-6 space-y-4 text-gray-700 sm:text-lg sm:leading-7">
+            <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+              <div class="relative">
+                <%= f.label :email, class: 'absolute left-0 -top-3.5 text-gray-600 text-sm peer-placeholder-shown:text-base
+            peer-placeholder-shown:text-gray-440 peer-placeholder-shown:top-2 transition-all peer-focus:-top-3.5
```

Comment:
while it is expected to have this design, putting the login button outside of the form is counterproductive. I'm open to learning if there is a way to achieve the same functionality with a redesign.

Reply:
Hi @lincoln1883,  you don't need to place the signup button close to the form, your design matches the wireframe good job so far 👍🏿

Reply:
Hello @prolajumokeoni and @lincoln1883  I think it needs to be above the same as the given design since the inputs style are also different not only the signup button so It would be great if he improved them to match the given design 🙏🏻


- And sure there are easy ways to do it, it is super easy to make it 🔥